.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 cache: bundler
 language: ruby
 rvm:
-  - 1.8.7
```

Comment:
If we're going to drop 1.8 support, we should just drop all the way up to 2.2 and make this Racc 2.0.  Unfortunately I recently had to do a release for 1.8 users because apparently they still exist.